packages/db/test/backup.spec.js

```diff
@@ -89,8 +89,56 @@ describe('backup', () => {
 
     assert(backups, 'backups created')
     assert.strictEqual(backups.length, 1, 'upload has a single backup')
-    assert(backups[0]._id, 'backup has id')
     assert(backups[0].created, 'backup has inserted timestamp')
     assert.strictEqual(backups[0].url, initialBackupUrl, 'backup has correct url')
   })
+
+  it('backup urls are unique', async () => {
+    await client.createUpload({
+      user: user._id,
+      contentCid: cid,
+      sourceCid: cid,
+      authKey: authKeys[0]._id,
+      type,
+      dagSize: dagSize,
+      name,
+      pins: [initialPinData],
+      backupUrls: [initialBackupUrl]
+    })
+
+    upload = await client.getUpload(cid, user._id)
+    assert(upload, 'second upload created')
+
+    const backups = await client.getBackups(upload._id)
+
+    assert.strictEqual(backups.length, 1, 'upload has a single backup')
+    assert(new Date(backups[0].created) < new Date(upload.updated), 'backup was created before the new upload')
+  })
+
+  it('can backup chunked uploads', async () => {
```

Comment:
Probably worth also checking if same URL is provided that it does not have both?

Can we also make this test be isolated? I see we rely on side effects from previous ones with uploads, but it would be better to have a different CID here and perform  `createUpload` function twice

Reply:
Great improvement! I made some changes. The `creates unique backup urls` tests for duplicated URLs.